cmake/Dependencies.common.cmake

```diff
@@ -314,7 +314,7 @@ if(BUILD_NVIMAGECODEC)
     add_definitions(-DNVIMGCODEC_DEFAULT_INSTALL_PATH=\"${NVIMGCODEC_DEFAULT_INSTALL_PATH}\")
 
     # Find the position of the substring
-    string(FIND "aarch64-linux-gnu" "${CMAKE_PREFIX_PATH}" SUBSTRING_POSITION)
+    string(FIND "${CMAKE_PREFIX_PATH}" "aarch64-linux-gnu" SUBSTRING_POSITION)
```

Comment:
Why is this needed to bump the version?

Reply:
this is another bug that I am trying to address. The Tegra codepath was detected for x86, and I found that the syntax for string find was not right here